espanol/historieta/dago.md

```diff
@@ -0,0 +1,3 @@
+Y cuando creian que nada pasaba, lo de siempre paso. Dago, el veneciano, llamo la atencion de la dama y de su celoso esposo.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.